core/libs/auth/auth.php

```diff
@@ -309,9 +309,9 @@ public static function get_active_identity()
      *
      * @return string
      */
-    public static function get($var = '')
+    public static function get($var)
```

Comment:
__[PHPMD]__

get accesses the super-global variable $_SESSION.


<!-- Comment by [SideCI](https://sideci.com) -->